settings.gradle

```diff
@@ -12,3 +12,4 @@ include 'ndbench-geode-plugins'
 include 'ndbench-janusgraph-plugins'
 include 'ndbench-sample-plugins'
 include 'ndbench-web'
+include 'ndbench-aws'
```

Comment:
If we are telling folks to add new modules for each environment, shall we update the Wiki or Readme with the corresponding instructions?

Reply:
These modules are specific to connect to their cloud environments, in this case, this is especially needed for Dynamo and Dax. Yes, I have added a ReadMe for this module, and this module is referenced in Dynamo modues. I will work on documentation update in followup PR